core/common/src/ConcurrentQuery.ts

```diff
@@ -128,6 +128,10 @@ export interface QueryOptions extends BaseReaderOptions {
    * Determine row format.
    */
   rowFormat?: QueryRowFormat;
+  /**
+   * Default to false. Enables experimental features to be used in the query
+   */
```

Comment:
"experimental features" is extremely vague. Do you document somewhere what that actually means?